golem/task/taskcomputer.py

```diff
@@ -747,6 +747,8 @@ def task_given(
                 cpu_limit=cpu_time_limit))
 
     def has_assigned_task(self) -> bool:
+        logger.debug(
+            f"Has assigned task? assigned_subtasks={self.assigned_subtasks}")

```

Comment:
better for performance to use native logger formatting
```suggestion
            "Has assigned task? assigned_subtasks=%r", self. assigned_subtasks)

```

Reply:
ah, it's not even about performance, afaik, but rather e.g. analysis of logs with tools like sentry ... will include that commit once the tests pass so as not to intefere with it now

Reply:
It's also about performance. If you format your string manually, it's always done, regardless of log level. With formatting in logger, it can avoid doing unnecessary work.